warehouse/README.md

```diff
@@ -38,3 +39,30 @@ docker-compose up -d
 
 Then use any db client to connect to `localhost:5431` using the db credentials
 in `.env`.
+
+## Data Organization
+
+This warehouse currently stores the cleaned and merged source data from Kids First studies which have been ingested using the ingest [library](https://github.com/kids-first/kf-lib-data-ingest) and [packages](https://github.com/kids-first/kf-ingest-packages).
+
+The cleaned source data tables are produced by ingest library's extract stage and the merged data tables are produced by the library's transform stage. Any column name in any of these tables will be a concept attribute from the library's [standard concept schema](https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/common/concept_schema.py).
+Upon ingest of a study, a new database named after the study's KF ID will be created with two schemas: 1) `Extract` and 2) `Transform`. The below figure illustrates the organization of ingested data:
+
+[Draw.io Source](https://drive.google.com/file/d/1IJGg6Dw9TeNzLzI-9fnUvo71Y6HL8UHN/view)
+
+![Data Organization](./static/img/data-organization.png)
```

Comment:
Can you change from `Kids First RDS` to `Kids First AWS`

Reply:
Oops, my bad. Fixed the diagram.